Mage/src/main/java/mage/abilities/effects/common/HideawayPlayEffect.java

```diff
@@ -4,26 +4,45 @@
 import mage.abilities.Ability;
 import mage.abilities.effects.OneShotEffect;
 import mage.cards.Card;
+import mage.cards.CardsImpl;
 import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
 import mage.game.ExileZone;
 import mage.game.Game;
 import mage.players.Player;
+import mage.target.TargetCard;
 import mage.util.CardUtil;
 
+import java.util.List;
 import java.util.UUID;
 
 /**
  * @author LevelX2
  */
 public class HideawayPlayEffect extends OneShotEffect {
 
+    private final boolean playOne;
+
     public HideawayPlayEffect() {
         super(Outcome.Benefit);
+        this.playOne = false;
         staticText = "you may play the exiled card without paying its mana cost";
```

Comment:
better to have this constructor call `this(false)` instead of duplicating logic

Reply:
Yeah, that's what I was gonna do then I forgot how to do it off the top of my head.